src/controller.js

```diff
@@ -128,6 +127,28 @@ export default ['$scope', '$element', function ($scope, $element) {
     }
   });
 
+  $scope.$watch("layout.qHyperCube.qDataPages[0].qMatrix[0][0].qText", function (newValue, oldValue) {
+    if (newValue !== oldValue) {
+      try {
+        // Create hypercube
+        getCube($scope.layout.qHyperCube.qDimensionInfo[0].qGroupFieldDefs[0]).then(function (cube) {
+          $scope.currentCube = cube;
+          createTrellisObjects();
+        });
+      }
+      catch (err) {
+        // Destroy existing session objects
+        if ($scope.sessionIds.length) {
+          for (var i = 0; i < $scope.sessionIds.length; i++) {
+            enigma.app.destroySessionObject($scope.sessionIds[i]).then(function (res) {
+            });
```

Comment:
Empty function body, remove if not needed.

Reply:
Removed it as its not needed and corrected at one more place in the code, where it was similar empty function.